script/core/deployers/BlastMangroveDeployer.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {BlastMangrove} from "@mgv/src/chains/blast/core/BlastMangrove.sol";
+import {MgvReader} from "@mgv/src/periphery/MgvReader.sol";
+import {MgvOracle} from "@mgv/src/periphery/MgvOracle.sol";
+import {IBlastMangrove} from "@mgv/src/chains/blast/IBlastMangrove.sol";
+import {Deployer} from "@mgv/script/lib/Deployer.sol";
+import {MgvReaderDeployer} from "@mgv/script/periphery/deployers/MgvReaderDeployer.s.sol";
+import {BlastLib} from "@mgv/src/chains/blast/lib/BlastLib.sol";
+import {MangroveDeployer} from "./MangroveDeployer.s.sol";
+
+contract BlastMangroveDeployer is MangroveDeployer {
```

Comment:
This pattern is new and I find it a bit weird, since there's no actual inheritance. The only purpose is to be able to reuse the tests. And since the deployment scripts are totally independent, I’d say the tests should be as well.

Alternatively, you could introduce proper inheritance such that `BlastMangroveDeployer` only implements the few required changes to the `MangroveDeployer` script: Deploying `BlastMangrove` instead of `Mangrove` and setting the Blast governor.

I think the second option would be easier to maintain, as it maximizes reuse + makes it clear that those are the only two differences for that deployment.